_includes/code/howto/configure-rq/rq-compression-v4.py

```diff
@@ -0,0 +1,68 @@
+# THIS FILE NEEDS TESTS
+
+# ==============================
+# =====  CONNECT =====
+# ==============================
+
+# START ConnectCode
+import weaviate, os
+import weaviate.classes.config as wc
+
+client = weaviate.connect_to_local(
+    headers={
+        "X-OpenAI-Api-Key": os.environ[
+            "OPENAI_API_KEY"
+        ]  # Replace with your OpenAI API key
+    }
+)
+
+client.is_ready()
+
+# END ConnectCode
+
+# ==============================
+# =====  EnableRQ =====
+# ==============================
+
+client.collections.delete("MyCollection")
+
+# START EnableRQ
+import weaviate.classes.config as wc
+
+client.collections.create(
+    name="MyCollection",
+    vectorizer_config=wc.Configure.Vectorizer.text2vec_openai(),
+    # highlight-start
+    vector_index_config=wc.Configure.VectorIndex.hnsw(
+        quantizer=wc.Configure.VectorIndex.Quantizer.rq()
+    ),
+    # highlight-end
+)
+# END EnableRQ
+
+# ==============================
+# =====  EnableRQ with Options =====
+# ==============================
+
+client.collections.delete("MyCollection")
+
+# START RQWithOptions
+import weaviate.classes.config as wc
+
+client.collections.create(
+    name="MyCollection",
+    vectorizer_config=wc.Configure.Vectorizer.text2vec_openai(),
+    vector_index_config=wc.Configure.VectorIndex.hnsw(
+        distance_metric=wc.VectorDistances.COSINE,
+        vector_cache_max_objects=100000,
+        # highlight-start
+        quantizer=wc.Configure.VectorIndex.Quantizer.rq(
+            bits=8,  # Number of bits, only 8 is supported for now
+            rescore_limit=200,
```

Comment:
I don't have this parameter in the versions of the client that I have. Maybe I'm looking at a different version to you? 🤔

Reply:
Removed it for now as it's not in the Python client, but expecting it to come back as both BQ and SQ have it.